TowersOfHanoi/src/codeunit/Codeunit50000.SolveTowersOfHanoi.al

```diff
@@ -0,0 +1,48 @@
+codeunit 50000 SolveTowersOfHanoi
+{
+    trigger OnRun()
+    begin
+    end;
+
+    var
+        Err: Label 'Maximum Tower size is %1';
+
+    procedure Solve(var Rec: record TowerTable; NoOfDiscs: Integer);
+    begin
+        //check if there is anything to process
+        if NoOfDiscs > MaxStrLen(Rec.TowerA) then begin
+            Message(Err, MaxStrLen(Rec.TowerA));
+        end
+        else begin
+            Rec.DeleteAll;

```

Comment:
BP = https://alguidelines.dev/docs/bestpractices/deleteall/

Reply:
no need for being end for a single line statement